package.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "rea-web",
+  "version": "1.0.0",
+  "description": "Web front-end for Registered Email Address Service",
+  "main": "dist/server.js",
+  "scripts": {
+    "build:server": "tsc && cp -r src/views dist/ && cp -r src/config dist/",
+    "build:assets": "gulp build",
+    "build": "npm run build:assets && npm run build:server",
+    "lint": "eslint 'src/*.ts' 'src/**/*.ts' 'test/src/**/*.ts'",
+    "lint:fix": "eslint 'src/*.ts' 'src/**/*.ts' 'test/src/**/*.ts' --fix",
+    "coverage": "nyc --check-coverage --reporter=text-summary npm run test",
+    "coverage:report": "nyc --reporter=lcov --reporter=text npm run test",
+    "test:coverage": "nyc npm test",
+    "test": "mocha --recursive test/src/**/*.ts",
+    "start": "npm run build && source ./dist/config/.env && node dist/server.js",
+    "start:watch": "nodemon"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint && npm test"
+    }
+  },
+  "author": "Dale Bradley <dbradley@companieshouse.gov.uk>",
+  "license": "MIT",
+  "dependencies": {
+    "@companieshouse/api-sdk-node": "^1.0.5",
```

Comment:
We may want to review these versions, either now or very soon, as there are later versions of some of these libs.

Reply:
Updated to latest CH library versions